backend/src/main/java/com/google/models/Alert.java

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.models;
```

Comment:
nit: also we can discuss this (and helen might have different opinions), but not sure if the generic com.google.models is too generic of a package, since these 'models' (Alert, MetricValue, etc) are specific to the project. Again, not really a coding/programming type of issue so open for discussion.

Reply:
i agree with you that `com.google.models` is a very generic package name. 

would `com.google.greyswan_models` be a better name? i feel like it is more specific, but it is also longer...